content/docs/getting-started/scaffolding-components/index.md

```diff
@@ -1,36 +1,36 @@
 ---
-title: Scaffolding Components in Backstage
+title: Creating a template
```

Comment:
What do you think about titling it `Creating a scaffolder template` to keep the word scaffolder in there for ease of searchability?